libs/designsystem/src/lib/scss/_global-styles.scss

```diff
@@ -55,6 +55,20 @@ ion-modal:first-of-type {
     }
   }
 
+  &.modal-card {
+    .modal-wrapper {
+      height: 100% !important;
+      border-top-left-radius: 0 !important;
+      border-top-right-radius: 0 !important;
+
+      @include media('>=medium') {
+        height: 647px !important;
```

Comment:
Consider extracting  `647px` to a variable, since it's somewhat a "magic number" (needs context)